client/my-sites/upgrades/checkout/privacy-protection.jsx

```diff
@@ -59,22 +59,27 @@ module.exports = React.createClass( {
 	},
 
 	render: function() {
-		var numberOfDomainRegistrations = this.getNumberOfDomainRegistrations(),
-			priceForPrivacyText,
+		const numberOfDomainRegistrations = this.getNumberOfDomainRegistrations(),
 			hasOneFreePrivacy = this.hasDomainPartOfPlan() && numberOfDomainRegistrations === 1;
+		let priceForPrivacyText,
+			privacyText,
+			freeWithPlan;
 
 		priceForPrivacyText = this.translate(
-			'Privacy protection keeps your information hidden when this domain is searched for in the public database for an extra %(cost)s / year.',
-			'Privacy protection keeps your information hidden when these domains are searched for in the public database for an extra %(cost)s per domain / year.',
+			'%(cost)s per year',
+			'%(cost)s per domain per year',
 			{
 				args: { cost: this.getPrivacyProtectionCost() },
 				count: numberOfDomainRegistrations
 			}
 		);
 
-		if ( hasOneFreePrivacy ) {
-			priceForPrivacyText = this.translate( 'Privacy protection keeps your information hidden when these domains are searched for in the public database. You can use it for no extra cost.' );
-		}
+		freeWithPlan = hasOneFreePrivacy &&
+			<span className="checkout__privacy-protection-free-text" ref="subMessage">
```

Comment:
Why the `ref` here? It doesn't seem to be actually referenced anywhere, so it's probably some delicious copy-pasta 🍝 :wink: